src/Compilers/CSharp/Portable/Symbols/Synthesized/GeneratedLabelSymbol.cs

```diff
@@ -9,9 +9,19 @@ namespace Microsoft.CodeAnalysis.CSharp.Symbols
 {
     internal sealed class GeneratedLabelSymbol : LabelSymbol
     {
+        private readonly string _name;
+
         public GeneratedLabelSymbol(string name)
-            : base(LabelName(name))
         {
+            this._name = LabelName(name);
```

Comment:
just a habit. I like being able to easily distinguish member accesses and local accesses, but `_` is good enough for that.